pkg/cloudprovider/types.go

```diff
@@ -57,6 +57,7 @@ type InstanceType interface {
 	// Note that though this is an array it is expected that all the Offerings are unique from one another
 	Offerings() []Offering
 	Architecture() string
+	AMIFamily() string
```

Comment:
This is a vendor neutral API. We do our best to keep AWS specific concepts in our cloud provider code.

Reply:
Removed.